sehersarin/MSCI342-Term-Project
# S1 T1 create stub for creating endpoint to sign up user

Scrum Board Ticket: S1 T1

Story: Story 1 Sign up using email

Description: Users will create an account for the web application through an email address. They will also need to provide other essential information including their:

1. First Name
2. Last Name
3. Type (student or worker)
4. UserID
5. Email
6. Phone number
7. Password (to be created by the user)

This information is captured through the front end UI of the web application, and this stub aims to create a public method with an agreed upon method name, input parameters, output format, which verifies the credentials that are input to complete account creation.

Testing Criteria: 
I have created and run a test using the Arrange, Act, Assert format which is included in the accountCreation.test.js file. See the instructions below.

Instructions to Test:
1. Checkout this branch
2. Navigate to the server/test/accountCreation.test.js
3. A test user is provided with the following data:
          Test user:
            type: 'student', //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
            studentId: 20764242,
            firstname: 'Victor',
            lastname: 'Hugo',
            email: 'victorhugo@gmail.com',
            phone: "6476442200" 

The credentials to test this test user are: 
email = 'victorhugo@gmail.com'
password = 'victorlovespoetry'

4. If the user returned matches the test user, the account Creation is successful.
